lib/src/mock_query.dart

```diff
@@ -128,6 +128,12 @@ class MockQuery extends Mock implements Query {
       } else {
         return false;
       }
+    } else if (arrayContainsAny != null) {
+      if (value is Iterable) {
+        return Set.from(value).containsAll(arrayContainsAny);
```

Comment:
According to the specs, we should be checking that value contains at least one element in arrayContainsAny, not all of them.

Reply:
Yep, my latest commit addresses this. Tests have been updated accordingly.